### DISCUSSION TITLE
Add projects to mobile app

### DISCUSSION THREAD

**Discussion by OJFord** [community/community#5609] [Mobile] [answered]
I'd like to be able to view and update projects in the Android app.

'update' meaning:
- move card to another column
- add 'note' card to a column
- edit note, or follow issue card into the issue to edit/comment on the issue

---

Personally, I primarily use projects as a single overarching project for lots of miscellaneous personal repos that are too small for me to ever think to check independently; it's more convenient to have one place with all the issues and where I can decide how to prioritise them in a cross-repo way.

It'd be great to be able to do this from the app. Right now for example, it's the weekend, haven't turned on my computer yet, but I'm thinking maybe I'll work on something today - no notifications steering me towards anything particular, so I wish I could check my 'Project' in order to decide which .._project_ to work on.

Thanks!

**Comment by aconsuegra** [comment, marked as answer]
Hi @OJFord, thanks for your feedback! We are currently planning the best way to bring Projects related features to the GitHub Mobile apps, stay tuned for some news in the near future!

**Reply by MonkeyBars3k** [reply]
Agreed this is a very desired feature +1!!

**Comment by candyho** [comment]
Hey👋just want to inform you we've just shipped [view and edit projects(beta) custom fields]( https://github.blog/changelog/2022-05-09-view-and-edit-projectsbeta-custom-fields-on-github-mobile/) to both iOS and Android last week! You can now edit custom text, number, date, and single-select fields for issues and pull requests that belong to the new projects experience from within the triage sheet. Try it out and let us know what you think! Stay tuned for more updates!

**Reply by Wladefant** [reply]
@OJFord @candyho I think this should be marked as the answer

**Reply by OJFord** [reply]
@Wladefant Why? That's a very small step towards this, and isn't even _one_ of the bullet points in OP.

**Reply by Wladefant** [reply]
Not really small and I meant instead of the answer above, so people can see, that something has already been done

**Reply by Wladefant** [reply]
@amrehman @candyho @eliperkins this discussion is now resolved and can be closed
